Update triage team internal Sentry documentation (#2097)
department-of-veterans-affairs/va.gov-team: platform/triage/Sentry.md

*  **Issue ownership rules** - assign issues to individuals or Sentry teams based upon the code path or url. Updates will be necessary as new features and apps are added. [example](http://sentry.vetsgov-internal/settings/vets-gov/projects/platform-api-production/ownership/)

* **Sentry Teams** [example](http://sentry.vetsgov-internal/settings/vets-gov/projects/platform-api-production/teams/) 

> * **Response Process** - It was decided to move away from the on call rotation schedule in relation to Sentry alert response processes in the #vfs-sentry-alerts Slack channel. The new approach involves all sentry team member monitoring of incoming alerts. If an alert requires investigation, a solution will be coordinated on a case by case basis between triage members.

Reviewer: ```suggestion
* **Response Process** - It was decided to move away from the on call rotation schedule in relation to Sentry alert response processes in the #vfs-sentry-alerts Slack channel. The new approach involves all triage team members monitoring Sentry for incoming alerts. If an alert requires investigation, a solution will be coordinated on a case by case basis between triage members.
```

Reviewer: Good to go after a few minor changes, thanks for updating this!
